### ISSUE TITLE
Button missing a text alternative - Commencement venue map marker

### ISSUE THREAD

**Issue by briand44** [uiowa/uiowa#9686]
# Motivation

Siteimprove is reporting "Button missing a text alternative". This is a WCAG 2.1 A compliance issue related to the map marker on the Commencement venue maps.

# Siteimprove info

**Technical doc**
https://alfa.siteimprove.com/rules/sia-r12

Siteimprove link to existing issue
- https://my2.siteimprove.com/Inspector/1005482/A11y/Page?pageId=51028001886&impmd=264ED37B65FA6186129334A508391970&conf=a+aa#/sia-r12/failed/aGthSkNscWs=;N2UzNDM5YzY=
- https://my2.siteimprove.com/Inspector/1005482/A11y/Page?pageId=47303686503&impmd=264ED37B65FA6186129334A508391970&conf=a+aa#/sia-r12/failed/anVUZjhoazQ=;ZjFjMmI2M2E=

Example
- https://commencement.uiowa.edu/venue/pappajohn-business-building
- https://commencement.uiowa.edu/venue/hancher-auditorium

**Button missing a text alternative**
Issue
4.1.2: Name, Role, Value
Buttons should always have a text label. The label can be inserted into the HTML — it does not need to be visible to pass this check.
